src/strings.json

```diff
@@ -26,7 +26,7 @@
         },
         "components": {
             "operational": "Operational",
-            "performanceIssues": "Performance Issues",
+            "performanceIssues": "Performance issues",
```

Comment:
Rein aus Konsistenzgründen macht es keinen Sinn, Issues hier klein zu schreiben. Entweder schreibt man das zweite Wort dann in allen Stati klein oder überall groß, wobei ich in diesem Fall sogar argumentieren würde, dass es sinvoller wäre, die Wörter als Titel groß zu schreiben.